rdkit/Chem/TorsionFingerprints.py

```diff
@@ -428,8 +433,8 @@ def CalculateTorsionWeights(mol, aid1=-1, aid2=-1, ignoreColinearBonds=True):
   # get shortest paths and calculate weights
   weights = []
   for bid1, bid2, nb1, nb2 in bonds:
-    if ((bid1, bid2) == (aid1, aid2) or
-        (bid2, bid1) == (aid1, aid2)):  # if it's the most central bond itself
+    if ((bid1, bid2) == (aid1, aid2)
```

Comment:
Was this a format change?  I find the previous more readable, but YMMV.

Do we have a python formatter that I should be using? :)

Reply:
yeah, that's just a format change.
I've been using yapf, because i think that's what @gedeck used a few years ago to reformat a bunch of the code